README.md

```diff
@@ -11,23 +11,6 @@ A kit takes care of initializing and forwarding information depending on what yo
 
 Please refer to installation instructions in the core mParticle Apple SDK [README](https://github.com/mParticle/mparticle-apple-sdk#get-the-sdk), or check out our [SDK Documentation](http://docs.mparticle.com/#sdk-documentation) site to learn more.
 
-
```

Comment:
Yes, this is no longer needed. It strictly doesn't belong in this PR, but figured we could probably throw it in there as it's a small standalone change. We now support dynamic linking, so the static instructions are obsolete.